toolchain/semantics/semantics_ir.cpp

```diff
@@ -5,13 +5,19 @@
 #include "toolchain/semantics/semantics_ir.h"
 
 #include "common/check.h"
-#include "llvm/Support/FormatVariadic.h"
-#include "toolchain/lexer/tokenized_buffer.h"
+#include "toolchain/semantics/node_kind.h"
+#include "toolchain/semantics/node_ref.h"
+#include "toolchain/semantics/node_store.h"
```

Comment:
There's going to be a close link between semantics_ir.cpp and semantics_ir.h: let's not duplicate #includes between them.